### DISCUSSION TITLE
求助目录章节标题如何改为不加粗

### DISCUSSION THREAD

**Discussion by zhangsipeng** [sjtug/SJTUThesis#962] [提问]
<img width="241" alt="c43b910e49269d8eac1a54e517ad032" src="https://github.com/sjtug/SJTUThesis/assets/74140050/4c4649b9-3361-4824-bee9-294a5e98a56e">
请问如何把上述 "第一章 绪论"这几个字不加粗，改为和目录中其他字体一样

**Comment by LogCreative** [comment]
Word 文档中也是加粗的：
<img width="761" alt="截屏2023-06-06 15 48 01" src="https://github.com/sjtug/SJTUThesis/assets/61653082/b645e301-2fcc-4d09-9740-d639d9704633">

**Comment by zhangsipeng** [comment]
好的，谢谢，那没啥问题，这个图书馆专家审核好像标准不是统一的QAQ
